openstf/model/fallback.py

```diff
@@ -0,0 +1,68 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+from pathlib import Path
+from datetime import datetime, timedelta, timezone
+
+import pandas as pd
+import structlog
+
+
+def generate_fallback(
+    forecast_input: pd.DataFrame,
+    load: pd.DataFrame,
+    fallback_strategy: str = "extreme_day",
+) -> pd.DataFrame:
```

Comment:
Maybe good to think about consistency for the naming here as well.
 
 We used to have a `create_forecast` task which ran the `create_forecast.create_forecast_pipeline()` which used the `prediction_model.make_forecast()` method to make a `forecast`.
 
 Now we are moving to some other/better? vocabulary, but we seem to also introduce some inconsistencies. We still have the `create_forecast` task, but we now use `predict_sklearn.predict_pipeline` to run the pipeline which calls a sklearn model `model.predict()` to make a `forecast`
 
 I do like generate here, but I think you should also update the other vocabulary to restore consistency.
 
 Starting at the bottom if we have `model.predict()` to make a regular forecast perhaps it makes sense to go for a similar interface and or vocabulary here as well. So `fallback_model.predict()` or `model.predict_fallback()` etc.

Reply:
after discussion, let's leave it as is for this pr